src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -36,7 +36,11 @@ public function __construct($documentRoot, $env, $address = null, $router = null
 
         $this->documentRoot = $documentRoot;
         $this->env = $env;
-        $this->router = $router ?: __DIR__.'/Resources/router.php';
+        $this->router = $router ? realpath($router) : __DIR__.'/Resources/router.php';
+
+        if (false === $this->router || !file_exists($this->router)) {
```

Comment:
I think that realpath returns false when the file dont not exist anyway. That'd means the file_exists check is useless, isn't it?